brave/brave-core
# Add unique index to contribution_info_publishers

Resolves https://github.com/brave/brave-browser/issues/7838

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Enable Rewards
- Claim promotion
- Make a contribution
- Ensure that contribution is deducted
- Ensure that data is correct in database (`contribution_info_publishers` table)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
